kubernetes/ingress/planetaryresponsenetwork.org.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: planetaryresponsenetwork-org-tls
+  labels:
+    use-azuredns-solver: "true"
+spec:
+  issuerRef:
+    name: letsencrypt-prod
+    kind: ClusterIssuer
+  secretName: planetaryresponsenetwork-org-tls
+  dnsNames:
+    - planetaryresponsenetworkorg
```

Comment:
The DNS names in this one are missing the "."

Reply:
Fixed in https://github.com/zooniverse/static/pull/159